distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1373,12 +1373,30 @@ type Table
          - If any column names in the new table are clashing, a
            `Duplicate_Output_Column_Names` is reported according to the
            `on_problems` setting.
-    @id_fields Widget_Helpers.make_column_name_vector_selector
+
+       ? Example Transpose Operation
+
+         Input Table `table`:
+
+            A | Name    | Country
+           ---|---------|---------
+            A | Example | France
+            B | Another | Germany
+
+         Result `table.transpose ['A'] 'Attribute' 'Value'`:
+
+            A | Attribute | Value
+           ---|-----------|---------
+            A | Name      | Example
+            A | Country   | France
+            B | Name      | Another
+            B | Country   | Germany
```

Comment:
THIS IS SO GREAT!

I used to not know what transpose does and I would have really appreciate this example back in the day.

I'd just suggest to maybe rename the column `A` as it clashes with the value `A` and while the context is rather clear, it can lead to some ambiguity. Maybe let's call the column `Id` or `X`?